transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SQLiteEventStore.java

```diff
@@ -286,6 +308,23 @@ public void close() {
     schemaManager.close();
   }
 
+  @VisibleForTesting
```

Comment:
Is this operation only for testing or is only the visibility adjusted ? Can we move it to the tests themselves ?

Reply:
It's only needed in tests, I replaced it with `@RestrictTo(TESTS)` which is more strict and warns for any use of the method in SDK code. It is problematic to move the code to the test itself as the implementation of the method uses internal state of the `store` that I'd rather not expose as public.